src/i18n/EnkiIntlProvider.tsx

```diff
@@ -25,16 +26,27 @@ export const EnkiIntlProvider = ({
   useEffect(() => {
     getMessages(selectedLocale).then((messages) => {
       setMessages(() => messages);
+      if (partnerCompany) {
+        getExtMessages(partnerCompany, selectedLocale)
+          .then((extMessages) => {
+            const combinedMessages = {
+              ...messages,
+              ...extMessages,
+            };
+            setMessages(() => combinedMessages);
+          })
+          .catch();
+      }
     });
-  }, [selectedLocale]);
+  }, [selectedLocale, partnerCompany]);
 
   return (
     <Suspense>
       {deferredMessages !== null && (
         <IntlProvider
           locale={selectedLocale}
-          messages={deferredMessages}
```

Comment:
This felt questionable, there is probably a reason for this that I don't know. But there were problems that when switching the language the translations would get almost always delayed until the next locale change (also sometimes noticeable in the demo env) and with this change it got solved

Reply:
I've had a look at this again now, and I can't reproduce any problems with deferredMessages when the Fintraffic feature flag is off, it's only an issue when the flag is on. Not clear why, I tried also removing other changes but result was the same. Would you like to have a conditional in there like `messages={partnerCompany ? messages : deferredMessages}`, so the previous way stays same for Entur in case there was some reason for deferredMessages?

Reply:
Like I said, I don't remember why it's there, and if it works without it then we can remove it.

Reply:
Apparently, after removing the `deferredMessages !== null`, this kind of log shows up 
<img width="691" alt="Screenshot 2024-06-11 at 16 11 43" src="https://github.com/entur/enki/assets/85493679/a09f368b-2684-410f-af34-6f81e649f6c1">